docs/install.md

```diff
@@ -8,17 +8,20 @@ If you run into any problems with the instructions below, check out [the Trouble
 
 	We recommend that you follow the step to install the `vagrant-hostsupdater` plugin, because if you don't then
 	you will need to manually add the hostnames for each WME site to your local hosts file.
-
-1. Clone this repository as a subdirectory of VVV's `www` folder, and name the folder `wordpress-meta-environment`
-	1. `cd vagrant-local/www` (where `vagrant-local` is the name of your VVV folder)
-	1. `git clone https://github.com/WordPress/meta-environment.git wordpress-meta-environment`
-
-	**It's important that you clone the repository to a folder named `wordpress-meta-environment`.** If you choose
-	a different name, then WME will break during the provision stage, unless you also manually edit the Nginx
-	configuration files for each site to reflect the name you chose.
+	
+	Make sure you have the latest version of VVV.
+
+1. In the `vagrant-local` folder edit `vvv-config.yml` and uncomment the following line:
+       
+       ```yml
+       #wordpress-meta-environment: https://github.com/WordPress/meta-environment.git
```

Comment:
Second Review:

I'd still like to see this PR updated so that the docs clearly state what needs to be changed.

The VVV `vvv-config.yml` file does not display this on a single line, it is displayed across two lines, the code we ask a user to change should match that of the source code, so per the original `vvv-config.yml` can we have this the same, i.e. two lines:

> https://github.com/Varying-Vagrant-Vagrants/VVV/blob/develop/vvv-config.yml#L28-L29
```yaml
  #wordpress-meta-environment:
  #  repo: https://github.com/WordPress/meta-environment.git
```